content/en/docs/concepts/configuration/secret.md

```diff
@@ -1244,83 +1227,36 @@ these pods.
 
 ## Information security for Secrets
 
-Although ConfigMap and Secret work similarly, Kubernetes applies some additional
-protection for Secret objects.
+Although ConfigMaps and Secrets work similarly, Kubernetes applies some
+additional protections to Secret objects.
 
-Secrets often hold values that span a spectrum of importance, many of which can
-cause escalations within Kubernetes (e.g. service account tokens) and to
-external systems. Even if an individual app can reason about the power of the
-Secrets it expects to interact with, other apps within the same namespace can
-render those assumptions invalid.
+Secrets often hold values that span a spectrum of importance. Many of these
+values can cause escalations within Kubernetes and to external systems. Even if
+an individual app can reason about the power of the Secrets it expects to
+interact with, other apps within the same namespace can render those assumptions invalid.
 
 A Secret is only sent to a node if a Pod on that node requires it.
-For mounting secrets into Pods, the kubelet stores a copy of the data into a `tmpfs`
-so that the confidential data is not written to durable storage.
-Once the Pod that depends on the Secret is deleted, the kubelet deletes its local copy
-of the confidential data from the Secret.
+For mounting secrets into Pods, the kubelet stores a copy of the data into a
+`tmpfs` so that the confidential data is not written to durable storage.
+When the Pod that depends on the Secret is deleted, the kubelet deletes its
+local copy of the confidential data from the Secret.
 
 There may be several containers in a Pod. By default, containers you define
-only have access to the default ServiceAccount and its related Secret.
-You must explicitly define environment variables or map a volume into a
-container in order to provide access to any other Secret.
+only have access to the default `ServiceAccount` object and its related Secret.
+To provide access to other Secret objects, you must explicitly map those objects
+to your containers.
 
 There may be Secrets for several Pods on the same node. However, only the
 Secrets that a Pod requests are potentially visible within its containers.
-Therefore, one Pod does not have access to the Secrets of another Pod.
+One Pod does not have access to the Secrets of another Pod.
 
 {{< warning >}}
-Any privileged containers on a node are liable to have access to all Secrets used
+Any privileged containers on a node might have access to all Secrets used
```

Comment:
```suggestion
Any privileged containers on a node can access all Secrets used
```

Reply:
I'm going to keep as might because we don't know the exact privileges of the containers (the intent wasnt "containers with privileged: true" -- is that what this implies?

Reply:
Yeah atleast I read it as "privileged: true" :)

Reply:
Might be good to explicitly call this out

```
When a container runs with `privileged: true`, it becomes a privileged container that can access all secrets on the host.
```